packages/atomic/project.json

```diff
@@ -1 +1,264 @@
-{}
+{
+  "namedInputs": {
+    "tests": ["{projectRoot}/src/**/*.{spec.{ts,tsx},spec.{ts,tsx}.snap}"],
+    "e2e": ["{projectRoot}/cypress/**"],
+    "stories": ["{projectRoot}/src/**/*.{?(*([^.]).)stories.{tsx,css},mdx}"],
+    "source": [
+      "{projectRoot}/stencil.config.ts",
+      "{projectRoot}/collection-manifest.json",
+      "{projectRoot}/tailwind.config.js",
+      "{projectRoot}/svg.transform.js",
+      "{workspaceRoot}/tsconfig.json",
+      "{projectRoot}/tsconfig.json",
+      "{projectRoot}/src/**/*([^.]).!({{spec.{ts,tsx},spec.{ts,tsx}.snap},{?(*([^.]).)stories.{tsx,css},mdx}})",
+      "{projectRoot}/stencil-plugin/**"
+    ]
+  },
+  "targets": {
+    "clean": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "rimraf dist docs loader www storybook-static src/components/search/atomic-search-interface/lang src/generated",
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:build:locales": {
+      "dependsOn": [],
+      "inputs": [
+        "{projectRoot}/scripts/create-generated-folder.mjs",
+        "{projectRoot}/scripts/split-locales.mjs",
+        "{projectRoot}/scripts/copy-dayjs-locales.mjs",
+        "{projectRoot}/src/locales.json"
+      ],
+      "outputs": [
+        "{projectRoot}/src/generated/availableLocales.json",
+        "{projectRoot}/src/generated/dayjs-locales-data.ts",
+        "{projectRoot}/src/components/search/atomic-search-interface/lang"
+      ],
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "node ./scripts/create-generated-folder.mjs",
+          "node ./scripts/split-locales.mjs",
+          "node ./scripts/copy-dayjs-locales.mjs"
+        ],
+        "parallel": false,
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:build:stencil": {
+      "dependsOn": ["^build", "cached:build:locales"],
+      "inputs": ["^source", "source"],
+      "outputs": [
+        "{projectRoot}/dist",
+        "{projectRoot}/www",
+        "{projectRoot}/docs",
+        "{projectRoot}/loader",
+        "{projectRoot}/src/components.d.ts"
+      ],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node --max_old_space_size=6144 ../../node_modules/@stencil/core/bin/stencil build",
```

Comment:
There's an unofficial Stencil executor we could use (see [`@nxext/stencil:build`](https://nxext.dev/docs/stencil/executors.html)), and this executor seems to just use the same config we already have, which would remove the need for referencing `../../node_modules` directly and probably ensure that `dev` works properly without messing with it so much.

The package itself doesn't seem very popular, but their monorepo which contains more executors has 350 stars: https://github.com/nxext/nx-extensions

I haven't tested it yet, since my concern for now wasn't to make `dev` scripts work and I didn't wanna add too much more to this PR.

Reply:
I'd favor using the core executor